### DISCUSSION TITLE
How to display errors in the browser ?

### DISCUSSION THREAD

**Discussion by Co-7** [eslint/eslint#18464] [Q&A]
I've done a little eslint config with version 8.57.0 but I can't understand why the errors aren't displayed in the browser and what I need to add to make this happen.

```js
import pluginVue from "eslint-plugin-vue";
import tsparser from "@typescript-eslint/parser";
import vueparser from "vue-eslint-parser";

export default [
  ...pluginVue.configs["flat/base"],
  {
    ignores: ["dist/**"],
    rules: {
      "vue/no-unused-vars": "error",
      "vue/multi-word-component-names": "error",
      "vue/component-definition-name-casing": ["error", "kebab-case"],
    },
    languageOptions: {
      parser: vueparser,
      parserOptions: {
        parser: tsparser,
      },
    },
  },
];

```

Like this : 

![image](https://github.com/eslint/eslint/assets/48591820/40ea568b-ca37-467a-8e4b-5e6d89adaebe)

**Comment by Rec0iL99** [comment]
Hi @Co-7, ESLint by default displays linting results in the CLI. If you want to display the results in browser then you can use the [`html`](https://eslint.org/docs/latest/use/formatters/#html) formatter.

[usage](https://eslint.org/docs/latest/use/command-line-interface#-f---format)

**Reply by Co-7** [reply]
I don't see how it works, when I do a `npm run dev` I don't run eslint, normally when I do a `npm run dev` it displays eslint errors in the browser I don't run anything myself.

package.json : 
```json
  "scripts": {
    "dev": "vite --host",
    "build": "vue-tsc && vite build",
    "preview": "vite preview",
    "lint": "eslint ."
  },
```

Even when an eslint error is created in the code, nothing is displayed in my cli

I only see errors in my Vscode and when I do an `npm run lint`.
